src/components/Dashboard/TodoList.vue

```diff
@@ -343,7 +343,11 @@
                 <!-- NB: blocks below are mutually exclusive, and order is important -->
 
                 <!-- this loading button pre-empts all buttons below -->
-                <template v-if="inProcessFiling === item.filingId">
+                <!-- special case to not show spinner for approved continuation in drafts -->
+                <template 
+                  v-if="inProcessFiling === item.filingId && 
+                    !(EnumUtilities.isTypeContinuationIn(item) && EnumUtilities.isStatusApproved(item))"
+                  >
```

Comment:
Why not change the logic for `inProcessFiling` in the code instead?

I think that, with this change, the 5x polling is still being done in the background. It doesn't need to be done for a continuation in authorization.

Reply:
Good point, I've moved the check into the `highlighTask` method since the `inProcessFiling` is just a simple number type. 

I also noticed the bug was occurring for continuation in filings with status `ACTIVE`. I'm not sure what other status types we can expect for continuation in filings, so I changed it to only display the spinner for `PENDING` status continuation ins. Let me know if that makes sense to do or not.

Reply:
Thanks for the change. Looks good to me.

Reply:
> I also noticed the bug was occurring for continuation in filings with status `ACTIVE`. I'm not sure what other status types we can expect for continuation in filings, so I changed it to only display the spinner for `PENDING` status continuation ins. Let me know if that makes sense to do or not.

The continuation in draft had a status of Active? That's unexpected. I think the normal flow is:
DRAFT → AWAITING_REVIEW → APPROVED → PENDING → PAID → COMPLETED

Basically, we only want the retry to work for the PENDING (ie, while we wait for PAID).

Ask if this doesn't make sense.

Reply:
Right my bad, I was looking at the wrong status field. I was looking at the one in `continationIn` rather than the `header`.

<img width="520" alt="Screenshot 2024-11-06 at 3 40 54 PM" src="https://github.com/user-attachments/assets/8d2b3f3a-37c0-4a60-96bb-32c25e2a1bfe">

That makes sense though, thanks for the clarification!